Code/Mantid/Framework/DataObjects/src/ReflectometryTransform.cpp

```diff
@@ -223,8 +229,27 @@ Mantid::API::MatrixWorkspace_sptr ReflectometryTransform::execute(
   return ws;
 }
 
+boost::optional<boost::shared_ptr<Mantid::DataObjects::TableWorkspace>>
+createTable(boost::optional<
```

Comment:
This function (and any others in this file) need DOxygen headers. I appreciate that this isn't all your work, but would be grateful if you could add missing DOxygen.